packages/lib-classifier/src/plugins/drawingTools/models/marks/Ellipse/Ellipse.js

```diff
@@ -0,0 +1,95 @@
+import { getParentOfType, types } from 'mobx-state-tree'
+import { Ellipse as EllipseComponent } from '@plugins/drawingTools/components/'
+import { EllipseTool } from '@plugins/drawingTools/models/tools'
+
+import Mark from '../Mark'
+
+const BUFFER = 24
+const DELETE_BUTTON_ANGLE = -45
+const MINIMUM_RADIUS = 5
+
+const EllipseModel = types
+  .model('EllipseModel', {
+    x_center: types.maybe(types.number),
+    y_center: types.maybe(types.number),
+    rx: types.optional(types.number, 0),
+    ry: types.optional(types.number, 0),
+    angle: types.optional(types.number, 0)
+  })
+  .views(self => ({
+    get coords () {
+      return {
+        x: self.x_center,
+        y: self.y_center
+      }
+    },
+
+    deleteButtonPosition (scale) {
+      const theta = DELETE_BUTTON_ANGLE * (Math.PI / 180)
+      const dx = ((self.rx + BUFFER) / scale) * Math.cos(theta)
+      const dy = ((self.ry + BUFFER) / scale) * Math.sin(theta)
+      return {
+        x: dx,
+        y: dy
+      };
+    },
+
+    get isValid () {
+      return self.rx - MINIMUM_RADIUS > 0
+    },
+
+    get tool () {
+      return getParentOfType(self, EllipseTool)
+    },
+
+    get toolComponent () {
+      return EllipseComponent
+    },
+    
+    get x () {
+      return self.x_center
+    },
+
+    get y () {
+      return self.y_center
+    }
+  }))
+  .actions(self => {
+
+    function initialDrag ({ x, y }) {
+      const rx = self.getDistance(self.x, self.y, x, y)
+      const angle = self.getAngle(self.x, self.y, x, y)
+      self.rx = rx
+      self.ry = rx * 0.0001
```

Comment:
Maybe a TODO comment here about switching this to configurable value from the lab?

Reply:
Let's add it as a configurable value if volunteers or project owners ask for it.

Reply:
This is not consistent with the decision in ADR 24. If we're going to do something different, i.e. really small values for defaults, then we should document that as the decision instead when, why, etc. I'd prefer we stay consistent with ADR 24 unless we have good reason to do something different.

Reply:
It can be set to 0. The rendering is a bit weird for a zero-radius shape, but you can still draw ellipses.

Reply:
Ok, I'd like a TODO comment here and an updated issue so that we know this is something to come back to. Defaulting to either a very small number or zero has some UX implications that I think Becky should comment on, but ultimately, the decision is to make this configurable, so there needs to be planned work at some point for updating the lab unless an argument is made to update the proposed ADR with an alternative proposal. I asked about really small defaults in the discussion, but the message is strongly none if possible and if they have to exist, make the project owners actively involved in setting them so they understand the effect.